spec/models/file_set/file_set_derivatives_spec.rb

```diff
@@ -5,7 +5,7 @@
   let(:file)        { build(:department_file, id: '1234') }
   let(:derivatives) { CurationConcerns::DerivativePath.derivatives_for_reference(file) }
 
-  before { LakeshoreTesting.reset_derivatives }
+  before(:each) { LakeshoreTesting.reset_derivatives }
```

Comment:
No, I'm not right here, :each doesn't solve this problem always. It will error sometimes (for the same reasons, an extra access-pdf leftover) but extremely inconsistently: a seed from running the whole suite with rake spec will *not* cause the error when running it with just the fileset_derivatives_spec.rb. 

I don't know how to fix this yet.